src/api.tsx

```diff
@@ -36,7 +36,32 @@ const get = <T extends {}>(url: string) => async (
 
 const getEvents = () => get<FRCEvent[]>('events')
 
-const getEvent = (eventKey: string) => get<FRCEvent>(`events/${eventKey}`)
+const getEvent = (eventKey: string) => async (
+  cb: (err: Error | null, data: FRCEvent | null) => any
+) => {
+  get<FRCEvent>(`events/${eventKey}`)((err: Error, data: FRCEvent) => {
+    if (err) {
+      cb(err, data)
+    } else {
+      if (data != null && data.matches != null) {
```

Comment:
Triple equals

Reply:
What if it is undefined? Or should I explicitly check for both with the triple equals?

Reply:
When will it be undefined?